CHANGELOG.md

```diff
@@ -22,6 +22,18 @@ This changelog follows the semantic versioning standard(https://semver.org)
 - N/A
 -->
 
+## 8.0.0 - 2025-09-15
```

Comment:
Please can you change this to 7.3.0? (While the change is significant, it still is backwards-compatible (nothing changed, only new things added)